### PR TITLE
Added onboarding route; info from onboarding json

### PR DESCRIPTION
**Pull request by 0xJungleMonkey**
Hi, pull request for fixing issue #26 .
1. Added onboarding route for tampadevs.com/onboarding 
2. Onboarding file template with placeholder.
3. Pull data from json file onboarding.json

### CODE REVIEW

File: src/_includes/layouts/onboarding.njk [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+nav: true;
```
Comment (manuelosorio) [inline src/_includes/layouts/onboarding.njk]: The layout isn't being used for the onboarding page and is safe to remove.

File: src/onboarding/index.njk [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+layout: layouts/base.njk
+nav: true
+title: Onboarding
+aos: false
+---
+{% block css %}
+  <link rel="stylesheet" href="https://unpkg.com/flickity@2/dist/flickity.min.css">
```
Comment (manuelosorio) [inline src/onboarding/index.njk]: Flickity might not be needed in this page unless there are plans for adding a slider.

**Review by manuelosorio — COMMENTED** [review]
Thank you for the pull request. I'll wait for @vincentntang or @chtzvt to take a look at the changes. Overall good first pull request.

File: src/onboarding/index.njk (hunk 2 above)
Comment (0xJungleMonkey) [reply src/onboarding/index.njk]: All right. I removed it. Thanks for the feedback so much!

**Review by manuelosorio — APPROVED** [review]
Looks good to me.

**Pull request merged**